### ISSUE TITLE
No Chinese Simplified in Setup

### ISSUE THREAD

**Issue by sanddudu** [Bionus/imgbrd-grabber#703]
By the way, I think it's better to present the latest version download link instead of JSON. It's would be more convenience to fetch update setup in about window.

**Comment by Bionus** [comment]
Indeed, I could add Chinese to the setup. I removed all non-latin languages after some issue with the Russian setup language. Since there's no more hurry now, I can look in more detail what was the issue and restore them. As well as using the chosen language during setup as Grabber's language on first run if available.

> By the way, I think it's better to present the latest version download link instead of JSON. It's would be more convenience to fetch update setup in about window.

What JSON do you mean?

![image](https://cloud.githubusercontent.com/assets/882719/20869962/946d1180-ba7e-11e6-9ab3-1736e1252d79.png)
